discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -934,8 +931,17 @@ def wait_and_kill_installer(self, nodes, host):
         selected_node.kill_installer()
 
 
-def get_api_vip_from_cluster(api_client, cluster_info: models.cluster.Cluster):
+def get_api_vip_from_cluster(api_client, cluster_info: Union[dict, models.cluster.Cluster], pull_secret):
+    import warnings
```

Comment:
why we have internal imports?

Reply:
Ticket is too large and very difficult to understand what you mean

Reply:
>why we have internal imports? Cause we are going to deprecate it?

Yes, The internal import is just because it will be deprecated soon.


>i think you should explain what we should use in this case

The function and solution for that case have not been determined yet. I might be on another module of as a classmethod within Cluster class.